WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -256,6 +308,64 @@ public function check_for_matches( $stackPtr ) {
 	} // End check_for_matches().
 
 	/**
+	 * Verify if the current token is one of the targetted functions as callback.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @param int $stackPtr The position of the current token in the stack.
+	 *
+	 * @return int|false
+	 */
+	public function check_for_callback_matches( $stackPtr ) {
+
+		$token_content = strtolower( $this->tokens[ $stackPtr ]['content'] );
+
+		// Check if the function is used as a callback.
+		if ( ! isset( $this->callback_functions[ $token_content ] ) ) {
+			return false;
+		}
+
+		$skip_to    = array();
+		$parameters = $this->get_function_call_parameters( $stackPtr );
+		$positions  = $this->callback_functions[ $token_content ];
+
+		foreach ( $positions as $position ) {
+
+			// Calculate the last argument if the position is negative.
+			if ( $position < 0 ) {
+				$position = count( $parameters ) + 1 + $position;
+			}
+
+			if ( ! isset( $parameters[ $position ] ) ) {
+				return false;
+			}
+
```

Comment:
A callback can be an array. In that case, you want to pass the stack pointer of the `T_ARRAY` or `T_SHORT_ARRAY_OPEN` to the `get_function_call_parameters()` method again.